addon/components/cfb-form-editor/question.js

```diff
@@ -173,6 +145,9 @@ export default Component.extend(ComponentQueryManager, {
   namespace: computed("calumaOptions._namespace", function() {
     return slugify(this.calumaOptions.getNamespace() || "") || null;
   }),
+  prefix: computed("namespace", function() {
+    return this.namespace ? `${this.namespace}-` : "";
+  }),
```

Comment:
Might be possible to replace some of the `${this.namespace}-` calls further down in the file with `this.get("prefix")`